src/modules/apps/e2e/useInterval.tsx

```diff
@@ -0,0 +1,41 @@
+import { useRef, useEffect } from 'react'
+
+export function useInterval(callback: () => Promise<void>, delay) {
+  const savedCallback = useRef<() => Promise<void>>()
+
+  useEffect(() => {
+    savedCallback.current = callback
+  }, [callback])
```

Comment:
I'm not sure this is necessary, can you test it without it?

Reply:
https://pt-br.reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects

Reply:
I think callback doesn't change, not sure though